src/util.rs

```diff
@@ -409,14 +409,24 @@ pub mod test_helpers {
     ) {
         let receipts = client.transact(checked_tx);
 
-        if let Receipt::Panic { id: _, reason, .. } = receipts.get(0).expect("No receipt") {
+        if let Receipt::Panic {
+            id: _,
+            reason,
+            contract_id,
+            ..
+        } = receipts.get(0).expect("No receipt")
+        {
             assert_eq!(
                 &expected_reason,
                 reason.reason(),
                 "Expected {}, found {}",
                 expected_reason,
                 reason.reason()
             );
+            match expected_reason {
+                PanicReason::ContractNotInInputs => assert!(contract_id.is_some()),
```

Comment:
Maybe we should also have tests that the contract id is set to an non-zero value?

Reply:
Checked `contract_id` for `LCD` opt code